core/domain/user_services.py

```diff
@@ -47,6 +47,9 @@
 
 
+# TODO(#10178): gae_id will be deprecated for UserSettings because
```

Comment:
Frame TODOs as imperatives. I.e.:

"Deprecate gae_id for UserSettings once we have verified that UserAuthModels exists for every user."

(No need for the last sentence.)

Reply:
Done.